config/aws/aws.go

```diff
@@ -28,6 +28,7 @@ type (
 		RoleARN         string `envconfig:"AWS_ROLE_ARN"`
 		SecretKey       string `envconfig:"AWS_SECRET_KEY"`
 		SessionToken    string `envconfig:"AWS_SESSION_TOKEN"`
+		Endpoint        *string // Must be overridden separately for each service
```

Comment:
To keep with the environment config convention, mind adding a `envconfig:"AWS_ENDPOINT"` struct tag to the field?

Also, let's add a proper comment that will pass the linter. Something like:

> Endpoint is an optional endpoint URL (hostname only or fully qualified URI) that overrides the default endpoint for a client. Leave the value as "nil" to use the default endpoint. Currently, only the gizmo SNS Publisher is using this value.